packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -600,16 +600,19 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         uint32 lastVotingRound = priceRequest.lastVotingRound;
         VoteInstance storage voteInstance = priceRequest.voteInstances[lastVotingRound];
 
-        uint256 totalVotes = voteInstance.results.totalVotes;
-        uint256 totalCorrectVotes = voteInstance.results.getTotalCorrectlyVotedTokens();
-        uint256 stakedAtRound = rounds[lastVotingRound].cumulativeStakeAtRound;
+        uint128 totalVotes = voteInstance.results.totalVotes;
+        uint128 totalCorrectVotes = voteInstance.results.getTotalCorrectlyVotedTokens();
+        uint128 stakedAtRound = rounds[lastVotingRound].cumulativeStakeAtRound;
         bool isGovernance = priceRequest.isGovernance;
 
-        (uint256 wrongVoteSlash, uint256 noVoteSlash) =
+        (uint128 wrongVoteSlash, uint128 noVoteSlash) =
             slashingLibrary.calcSlashing(stakedAtRound, totalVotes, totalCorrectVotes, requestIndex, isGovernance);
 
-        uint256 totalSlashed =
-            ((noVoteSlash * (stakedAtRound - totalVotes)) + (wrongVoteSlash * (totalVotes - totalCorrectVotes))) / 1e18;
+        uint128 totalSlashed =
+            uint128(
+                ((noVoteSlash * (uint256(stakedAtRound) - totalVotes)) +
+                    (wrongVoteSlash * (uint256(totalVotes) - totalCorrectVotes))) / 1e18
+            );
```

Comment:
An alternative to avoid so many lines would be to return `uint256` from `calcSlashing` in the slashing library, but that would be bit less consistent.